src/chrome/content/rules/Cheatbook.de.xml

```diff
@@ -0,0 +1,11 @@
+<!--
+	Invalid certificate:
+	- blog.cheatbook.de
+-->
+
+<ruleset name="Cheatbook.de">
```

Comment:
This rule needs the mixedcontent platform attribute. Some pages load CSS over http.